src/test/java/de/tum/in/www1/artemis/TextExerciseIntegrationTest.java

```diff
@@ -73,13 +79,17 @@ public void submitEnglishTextExercise() throws Exception {
 
     @Test
     @WithMockUser(value = "instructor1", roles = "INSTRUCTOR")
-    public void deleteTextExerciseWithSubmissionWithTextBlocks() throws Exception {
+    public void deleteTextExerciseWithSubmissionWithTextBlocksAndClusters() throws Exception {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/gh/ls1intum/Artemis/pullRequest?prid=5968855)

Reply:
See f9e39e2dbe10ffa13483affe2de6359ac2244213